accounts/service_test.go

```diff
@@ -829,7 +829,9 @@ func TestAccountService(t *testing.T) {
 			errFunc := func(err error) {
 				lndMock.mainErrChan <- err
 			}
-			service, err := NewService(t.TempDir(), errFunc)
+			store, err := NewBoltStore(t.TempDir(), DBFilename)
```

Comment:
Same here if not already closed in the `validate` func.

Reply:
same comment as above